### PR TITLE
Improved main views on mobile devices

### PR DESCRIPTION
**Pull request by mllocs**
1b099e54-52eb-47ac-8520-ebd2d996677e

- Improved the main views on small devices. 
- Added a filter on the cover image to look better with the new brand blue. 
- Used bootstrap's Grid system to improve responsiveness and fix inconsistent paddings.

### CODE REVIEW

**Comment by mllocs** [comment]
[Clean up mobile views](https://trello.com/c/TlOegNDv/111-clean-up-mobile-views)

**Comment by enricostano** [comment]
Deployed on https://staging.timeoverflow.org/

@mllocs could you please check that everything is OK? Thanks!

File: app/views/application/_navbar.html.erb [hunk 1]
```diff
@@ -44,21 +44,19 @@
 <% if current_user && current_organization %>
 <nav role="navigation">
   <div class="container-fluid">
```
Comment (markets) [inline app/views/application/_navbar.html.erb]: @mllocs I think this should be "container", with "container-fluid" looks weird on desktop:

captura de pantalla 2018-06-05 a las 18 44 25

File: app/views/application/_navbar.html.erb [hunk 2]
```diff
@@ -44,21 +44,19 @@
...
-                <%= render 'application/menus/organization_reports_menu' %>
-                <%= render 'application/menus/organization_statistics_menu' %>
-                <%= render 'application/menus/organization_listings_menu' %>
-              <% end %>
-          </ul>
-        </div>
+    <div class="row">
+      <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+        <ul class="nav nav-pills actions-menu">
+            <%= render 'application/menus/user_list_link' %>
```
Comment (markets) [inline app/views/application/_navbar.html.erb]: Can we fix indentation here?

File: app/assets/stylesheets/application.css.scss [hunk 3]
```diff
@@ -418,7 +430,11 @@ form .checkbox input[type="checkbox"] {
 }
 
 .navbar-static-top {
-  background-color: $palette-turkey;
+  background-color: rgba(39, 151, 175, 0.9); // @palette-turkey with opacity
```
Comment (sauloperez) [inline app/assets/stylesheets/application.css.scss]: can't this really be done in SCSS?

File: app/assets/stylesheets/application.css.scss [hunk 4]
```diff
@@ -567,6 +581,7 @@ form .checkbox input[type="checkbox"] {
     font-weight: 500;
     letter-spacing: -0.107rem;
     margin-top: 1rem;
+    text-shadow: 1px 1px #333;
```
Comment (sauloperez) [inline app/assets/stylesheets/application.css.scss]: this deserves to be in a var IMO

**Comment by enricostano** [comment]
We'll deploy this to staging again once @mllocs solves pending questions.

**Comment by mllocs** [comment]
@markets @enricostano @sauloperez Thanks guys! I just pushed come commits addressing your comments 😊

File: app/assets/stylesheets/application.css.scss [hunk 5]
```diff
@@ -418,7 +435,7 @@ form .checkbox input[type="checkbox"] {
 }
 
 .navbar-static-top {
-  background-color: $palette-turkey;
+  background-color: rgba($palette-turkey, 0.9);
```
Comment (mllocs) [inline app/assets/stylesheets/application.css.scss]: @sauloperez I didn't know this was possible :D

**Comment by enricostano** [comment]
Deployed to https://staging.timeoverflow.org

**Comment by mllocs** [comment]
@enricostano Looks pretty good enough ™️ ®️

**Pull request merged**